packages/validator/src/services/block.ts

```diff
@@ -106,7 +106,7 @@ export class BlockProposingService {
         this.metrics?.blockProposingErrors.inc({error: "publish"});
         throw extendError(e, "Failed to publish block");
       });
-      this.logger.info("Published block", {...logCtx, graffiti, ...block.debugLogCtx});
+      this.logger.info("Published block", {...logCtx, graffiti: graffiti.substring(0, 32), ...block.debugLogCtx});
```

Comment:
This isn't quite right, because some characters aren't just a single byte each.
Eg:
```ts
new TextEncoder().encode("🚀")
// Uint8Array(4) [ 240, 159, 154, 128 ]
```

Reply:
good point, we could still add a helper function to properly trim the graffiti output but I think it is not worth just for the log message